### DISCUSSION TITLE
Filtering out "My feed"

### DISCUSSION THREAD

**Discussion by Jenbn** [dailydotdev/daily#1041] [Feature Requests]
"My feed" is a great feature, though I think is lacking a filter system for the saved tags. 

Scenario: a user has selected a total of 17 tags which corresponds to 4 main tags (e.g: testing, security & privacy, fundamentals, devops) based on her/his overall interested. However, at certain moments this user might be interested in reading only about one or more main tags, for example security & privacy but want to have the previous tags saved. 

Currently, in order to do so, the user needs to clear all the other tags and select them again, one by one, manually. 

A feature that allows users to have such freedom and control of their own feeds, it's something it will be nice to have.

**Comment by idoshamun** [comment]
Hi @Jenbn,

As a temporary solution, I'd suggest visiting the tag feeds: `https://app.daily.dev/tags/<tag>`
Let me know if it works
